Source/Mosa.Compiler.Framework/Delegate.cs

```diff
@@ -11,14 +11,14 @@ namespace Mosa.Compiler.Framework
 	/// <summary>
 	/// Patches delegates
 	/// </summary>
-	public static class DelegatePatcher
+	public static class Delegate
```

Comment:
this name change seems likely to create confusion with System.Delegate, would recommend keeping the `DelegatePatcher` name

Reply:
Agreed!

Reply:
Check.